test/src/code/code_test.dart

```diff
@@ -49,7 +49,7 @@ void main() {
       }
     });
 
-    test('parses lines', () {
+    test('Parses lines', () {
```

Comment:
other test descriptions are not capitalized. Is there a reason for this specific description to be capitalized in this PR?

Reply:
https://medium.com/@alexey.inkin/how-to-name-and-group-tests-in-a-dart-file-553e194bd8af